docs/archive-data.md

```diff
@@ -0,0 +1,314 @@
+---
+id: archive-data
+title: How to archive data
+sidebar_label: Archive data
+---
+
+Data that is generated from a Workflow, such as Event Histories,  persists in normal Temporal storage per the configured retention period. The retention period exists so that the persistence store is not overwhelmed by the accummulation of Workflow data over time. To preserve Workflow data indefinitely, you can use the Archival feature which automatically moves the data from the normal persistence storage to long-term storage after the Workflow retention period expires.
+
+:::info
+
+**Archival is currently disabled by default.**
+- Archival is disabled when running Temporal via `docker-compose` and can not be enabled.
```

Comment:
I think you could just say "Archival is not supported when running via `docker-compose`